usaspending_api/references/models.py

```diff
@@ -462,122 +462,117 @@ class LegalEntity(DataSourceTrackedModel):
 
     individuals
     '''
-    business_categories = ArrayField(models.TextField(), default=list)
+    business_categories = ArrayField(models.TextField(), default=list, null=True)
```

Comment:
Why is this allowed to be null? I think we ran into issues in the past where business_categories was `null` instead of an empty list.

Reply:
In the actual database, roughly 2.4% of legal_entity records are NULL and 6.7% are '{}'.  This was just changed to match the actual data (obviously).  We can change the database to not allow nulls, but we'll have to track down how this column is populated and ensure no more NULLs are inserted.  Who decides if this is a new/separate issue or part of DEV-1668?

Reply:
Added [DEV-1759](https://federal-spending-transparency.atlassian.net/browse/DEV-1759) to deal with `business_entities` issue separately.  @dpb-bah ok to sign off on this pull request as is?